### ISSUE TITLE
Best known clique numbers in readme.md do not match the DIMACS reference

### ISSUE THREAD

**Issue by leowonglaw** [shah314/clique#3]
I think that the [readme](https://github.com/shah314/clique/blob/master/README.md) needs to be updated because it's best known clique numbers do not match the [DIMACS reference](https://iridia.ulb.ac.be/~fmascia/maximum_clique/DIMACS-benchmark).

For example:
- C1000.9: 67 to 68
- C2000.9: 75 to 80

Thanks for the awesome work

**Comment by shah314** [comment]
Yes, this work was done several years back, so the best known cliques may have improved (for which the optimum is not known). Thanks.
